src/main/resources/templates/user/editPrefs.html

```diff
@@ -0,0 +1,200 @@
+</head>
+<body>
+    <!--Taskbar-->
+    <!--TODO: ACC/LOGIN redirects-->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">Wayfinder</a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                <li class="nav-item">
+                    <a class="nav-link active" aria-current="page" href="#">Home</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Temp</a>
+                </li>
+            </ul>
+            <ul class="navbar-nav ms-auto mb-2 mb-lg-0" style="padding-right: 25px;">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-mdb-toggle="dropdown" aria-expanded="false">UserID </a>
+                    <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown" style="min-width: 20px;">
+                        <li>
+                            <!--REDIRECT TO ACC SETTINGS-->
+                            <a class="dropdown-item" href="#">Account Settings</a>
+                        </li>
+                        <li>
+                            <!--REDIRECT TO LOGIN PAGE-->
+                            <a class="dropdown-item" href="#">Log out</a>
+                        </li>
+                    </ul>
+                </li>
+            </ul>
+          </div>
+        </div>
+    </nav>
+
+
+    <!--Back button container-->
+    <!--TODO: add redirect to landing page-->
+    <div class="container-fluid d-flex">
+        <form action="/" method="post">
+            <div class="d-grid gap-2" style="padding-top: 10px;">
+                <input class="btn btn-secondary" id="addPrefsNewUser" type="submit" value="Return to home"></input>
+                <p style="font-style: italic; font-size: small;">Changes will not be saved</p>
+            </div> 
+        </form>
+    </div>
+    <!--Add prefs container-->
+    <div class="container-fluid d-flex justify-content-center">
+        
+        <div class="col-md-4 m-sm-4" id="formInputsCont">
+            <!--ADD CORRECT FORM PATH LATER-->
+            <form action="/" method="post" class="errorCheck" novalidate>
+                <h2 class="row justify-content-center">Edit Preferences</h2>
+                <div class="container p-1">
+                    <label for="access">Accessibility</label>
+                    <select class="form-select" id="access" name="access" required th:value="${user.access}">
```

Comment:
This won't select values that you want to be selected,please see my as2 for reference